packages/augur-sdk/src/state/getter/Users.ts

```diff
@@ -1016,74 +1126,13 @@ export class Users {
     db: DB,
     params: t.TypeOf<typeof Users.getTotalOnChainFrozenFundsParams>
   ): Promise<UserTotalOnChainFrozenFunds> {
-    if (!params.universe && !params.account) {
+    if (!params.universe || !params.account) {
```

Comment:
Error message should say `requires a 'universe' and 'account'`, not or

Reply:
good one. updated.